src/components/Tabs.tsx

```diff
@@ -0,0 +1,37 @@
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab?: string;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab }) => {
+  const isSelected = (tab: Tab) => tab.id === selectedTab;
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({ 'is-active': isSelected(tab) })}
+            >
+              <Link to={isSelected(tab) ? '/tabs' : `/tabs/${tab.id}`}>
```

Comment:
The Link should navigate to the tab content by using the tab.id regardless of whether the tab is currently selected or not.

Reply:
Ok makes sense, -fixed